js/otlp/tracer.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const ACCESS_TOKEN = process.env.LS_ACCESS_TOKEN;
+const SATELLITE_URL = process.env.LS_SATELLITE_URL || '127.0.0.1:55678';
+
+const opentelemetry = require('@opentelemetry/api');
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const { SimpleSpanProcessor } = require('@opentelemetry/tracing');
+const { CollectorExporter } = require('@opentelemetry/exporter-collector');
+
+const EXPORTER = process.env.EXPORTER || '';
+
+module.exports = (serviceName) => {
+    const provider = new NodeTracerProvider();
+
+    const address = '127.0.0.1:55678';
+    const exporter = new CollectorExporter({
+        serviceName: serviceName,
+        url: SATELLITE_URL,
+        headers: { "lightstep-access-token": ACCESS_TOKEN },
```

Comment:
Node only supports gRPC at this time and does not have a headers option (the browser variant does). For Node you'll need to set `metadata` and `credentials`. For example:

```javascript
const grpc = require('grpc);

const metadata = new grpc.Metadata();
metadata.set('lightstep-access-token', ACCESS_TOKEN);

const exporter = new CollectorExporter({
  serviceName: serviceName,
  url: SATELLITE_URL,
  credentials: grpc.credentials.createSsl(),
  metadata,
});


```